### PR TITLE
Prepare CODEOWNERS before activating branch protection

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -3,68 +3,52 @@
 
 # Only files explicitly mentioned are guarded by code owners (no * rule).
 
-CODEOWNERS   @gerolf-da @remyhaemmerle-da
-LATEST       @gerolf-da @bame-da
+CODEOWNERS   @basvangijzel-DA @dylant-da @gerolf-da @paulbrauner-da @remyhaemmerle-da 
+LATEST       @basvangijzel-DA @dylant-da @gerolf-da @paulbrauner-da @remyhaemmerle-da
```
Comment (basvangijzel-DA) [inline CODEOWNERS:7]: should be sdk/LATEST

File: CODEOWNERS [hunk 2]
```diff
@@ -3,68 +3,52 @@
 
 # Only files explicitly mentioned are guarded by code owners (no * rule).
 
-CODEOWNERS   @gerolf-da @remyhaemmerle-da
-LATEST       @gerolf-da @bame-da
+CODEOWNERS   @basvangijzel-DA @dylant-da @gerolf-da @paulbrauner-da @remyhaemmerle-da
```
Comment (basvangijzel-DA) [inline CODEOWNERS:6]: Can we create a new group daml-change-owners for this?

File: CODEOWNERS (hunk 1 above)
Comment (basvangijzel-DA) [inline CODEOWNERS:7]: Similarly, can we create a new group, daml-release-owners for this?

Comment (remyhaemmerle-da) [reply CODEOWNERS:7]: I imagine we have to ask @nycnewman for creating a @digital-asset/daml-release-owners group. 
@nycnewman can you do that ?

Comment (basvangijzel-DA) [reply CODEOWNERS:7]: I've done it. (See https://github.com/orgs/digital-asset/teams/daml-release-owners/members to manager and https://github.com/orgs/digital-asset/teams/ to create new teams.)

Comment (remyhaemmerle-da) [reply CODEOWNERS:7]:
```suggestion
/sdk/LATEST @digital-asset/daml-release-owners
```

File: CODEOWNERS [hunk 3]
```diff
@@ -3,68 +3,52 @@
 
 # Only files explicitly mentioned are guarded by code owners (no * rule).
 
-CODEOWNERS   @gerolf-da @remyhaemmerle-da
-LATEST       @gerolf-da @bame-da
+CODEOWNERS   @basvangijzel-DA @dylant-da @gerolf-da @paulbrauner-da @remyhaemmerle-da 
+/sdk/LATEST @digital-asset/daml-release-owners
```
Comment (remyhaemmerle-da) [inline CODEOWNERS:7]:
```suggestion
/sdk/LATEST @digital-asset/daml-release-owners 
```

**Review by basvangijzel-DA — APPROVED** [review]

**Pull request merged**